src/languages/en.ts

```diff
@@ -668,6 +668,8 @@ export default {
         hold: 'Hold',
         holdRequest: 'Hold Request',
         unholdRequest: 'Unhold Request',
+        heldRequest: (comment) => `held this money request with the comment: ${comment}`,
+        unheldRequest: 'unheld this money request',
```

Comment:
Please check https://expensify.slack.com/archives/C049HHMV9SM/p1708396751518539
`money` word might not be needed

Reply:
Updated, thanks!